facia/app/AppLoader.scala

```diff
@@ -77,7 +77,7 @@ trait AppComponents extends FrontendComponents with FaciaControllers with FapiSe
   )
 
   val frontendBuildInfo: FrontendBuildInfo = frontend.facia.BuildInfo
-  override lazy val httpFilters: Seq[EssentialFilter] = wire[CommonFilters].filters ++ wire[PreloadFilters].filters
+  override lazy val httpFilters: Seq[EssentialFilter] = wire[CommonFilters].filters
```

Comment:
I've removed `wire[CommonFilters].filters`, because having it here means any HTML response gets a `Link` header added with all Frontend's static assets, including for DCR-rendered fronts where we don't want to preload Frontend assets.